src/Symfony/Component/Cache/Adapter/AbstractAdapter.php

```diff
@@ -67,6 +68,24 @@ function ($deferred, $namespace, &$expiredIds) {
         );
     }
 
+    public static function createDefault($namespace = '', $defaultLifetime = 0, $nonce = null, $directory = null, LoggerInterface $logger = null)
+    {
+        $fs = new FilesystemAdapter($namespace, $defaultLifetime, $directory);
+        if (null !== $logger) {
+            $fs->setLogger($logger);
+        }
+        if (!ApcuAdapter::isSupported()) {
+            return $fs;
+        }
+
+        $apcu = new ApcuAdapter($namespace, $defaultLifetime / 5, $nonce);
+        if (null !== $logger) {
+            $apcu->setLogger($logger);
+        }
+
+        return new ChainAdapter(array($apcu, $fs));
```

Comment:
factory moved to FrameworkBundle (I don't like creating/loading yet another utility class just for the factory)